musicbot/bot.py

```diff
@@ -49,6 +47,13 @@
 load_opus_lib()
 
 log = logging.getLogger(__name__)
+log.warning("Deleting logs older than 1 week")
+
+# Logs folder path
+path = 'logs/'
+# searches for files that are older than 7 days and deletes them
+result = [os.remove(file) for file in (os.path.join(path, file) for path, _, files in os.walk(path) for file in files) if os.stat(file).st_mtime < time() - 7 * 86400]
```

Comment:
This should be a config option, rather than default, permanent behavior. Some users may, even if unnecessary, find it important to keeps older logs to find a song from a user again, or if they want to keep logs as a result of tracking a rare issue